python/ql/src/semmle/python/frameworks/Stdlib.qll

```diff
@@ -910,6 +910,26 @@ private module Stdlib {
    */
   private string pathlibPathMethodExport() { result in ["as_posix", "as_uri"] }
 
+  /**
+   * Flow for type presering mehtods.
+   */
+  private predicate typePreservingCall(DataFlow::Node nodeFrom, DataFlow::Node nodeTo) {
```

Comment:
This is a very generic-sounding name (and QLDoc comment) for something that's specific to `pathlib`. Perhaps something with `pathlib` in the name would be better?
(Also, two typos: `preserving methods`)

Reply:
Yeah, I think I had the generalisation of this in my head...

Reply:
I created [an issue](https://github.com/github/codeql-python-team/issues/357) for this now.

Reply:
Alas, the typo survived. (And the name, given that it lives immediately inside the `Stdlib` module could really use some disambiguation.)